docs/CHANGELOG.md

```diff
@@ -6,12 +6,128 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## DevilutionX 1.2.0 (unreleased)
 ### Features
+- Hellfire support
+- Shareware support
+- Fully migrate saves between Diablo and Hellfire
+- Document [ini-file options](https://github.com/diasurgical/devilutionX/wiki)
+- Support file paths longer than 259 characters
+- Allow for up to 99 save games of each type
+- Much lower memory usage (now in line with the original)
+- Performance improvements
+- Gold picked goes to stack even when inventory is open
+- Pepin automatically heals
+- Reorganize ini-file
+- Show other players on the map
+- Widescreen menus
+- Scroll subtitles at the same speed as narration reader
+- Preserve list position when buy and selling items in stores
+- Use belt via the numpad
+- Close panels when pressing <kbd>ESC</kbd>
+- Shift-click to spend all stat points
+- Shift-click to clear readied spell
+- Shift-click consumables to move them between the belt and inventory
+- Shift-click equipment equip/unequip them
+- Preserve hotkeys and active spell across games
+- Set default active spell for new heroes
+- Apply sound volume to videos
+- DIABDAT.MPQ may now be upper case
 - Improved gamepad support:
   - D-Pad / Left stick navigation now works in all menus and is more responsive.
   - START + SELECT now opens the main menu.
   - Right stick mouse emulation has been improved.
   - Gamepads plugged in while the game is running can be used.
   - All connected gamepads can be used (previously, only the first one).
+- Add `--ttf-dir` and `--ttf-name` to allow for specifying a different UI font
+
+### Optional improvements
+- Improved transparency
+- Experience bar
+- Enemy health bar
+- Auto equip all item types
+- Auto pickup gold
+- Run in town (also in multiplayer)
+- Disable friendly fire
+- Show monster type
+- Disable walking sounds
+- No quest randomization
+- Adria refills mana
+- Barbarian and Bard heroes in Diablo
+- Controller mapping
+- Configurable network port
+- Support for portable installs
+
+### Platforms
+- Switch Windows releases to 64bit
+- Added support for PlayStation Vita
+- Added support for Nintendo 3DS
+- Added support for DragonFly BSD
+- Added AppImage for Linux
+
+### Bugfixes
+- Incorrect graphics under cave doors in multiplayer
+- Catacombs doors never having traps
+- Dragging on touch devices
+- Mouse not releasing when exiting the window
+- Multiplayer host crashing after beating the game
+- Minor color issues in some videos
+- Command-line output missing on Windows
+- Correct visuals and function of the joining-game-dialog
+- Mouse jumping to the top of the windows when using the menu
+
+### Original Diablo bugs
+- All bugs fixed in Hellfire 1.01
+- Some unique items morphing on a new game
+- Monsters being unable to properly see the location of firewalls
+- Fix more stability issues
+- Gold stacks are correctly topped when in dungeon
```

Comment:
What does this mean?

Reply:
The stack is toped to 5000/10000 before a new one is started, instead of 4813 when the next pickup is 188 etc.

Let me know if you can come up with a better way to word it :)

Reply:
```suggestion
- Gold piles will be filled to their max, before starting a new one
```